action.yml

```diff
@@ -14,6 +14,8 @@ inputs:
   devbox-version: # version of the devbox cli
     description: 'Specify devbox CLI version you want to pin to. Default to latest'
     default: ''
+  sha256-checksum: # the expected SHA256 checksum of the devbox binary.
+    description: 'Specify an explicit checksum for the devbox binary. For extra security on top of the existing checks in the devbox launch script'
 
 runs:
   using: "composite"
```

Comment:
Sorry it wouldn't let me comment on line 48, but if you change it to 
```
latest_url="https://releases.jetpack.io/devbox/stable/version"
```
It might fix one of the CI

Reply:
I made this change and I also added in an option to disable the CLI cache primarily for running these SHA tests. By default the CLI cache will be on, so parity should be maintained w/r/t the behavior here.